core/src/test/java/feign/client/DefaultClientTest.java

```diff
@@ -96,4 +99,20 @@ public void canOverrideHostnameVerifier() throws IOException, InterruptedExcepti
     api.post("foo");
   }
 
+  @Test
+  public void testWsdlApi() throws InterruptedException {
```

Comment:
Similar comment as above.  This test is not about a `wsdl` but that that the request template can correctly generate a url where there exists a parameter with no value.